### DISCUSSION TITLE
Creating a WhatsApp Group for Sharing Knowledge on Trading with ib_async

### DISCUSSION THREAD

**Discussion by Amir-Inbar** [ib-api-reloaded/ib_async#48] [Ideas]
Hi everyone,

I'm interested in forming a WhatsApp group dedicated to sharing knowledge and experiences on trading using ib_async. This group will be a platform for developers and traders to discuss strategies, troubleshoot issues, and share insights related to Interactive Brokers API and the ib_async library.

Is it okay to share the WhatsApp group link here?

Thanks!

**Comment by vasiche** [comment]
There is already a discord (Mel's discord).  Are you in it?
-------- Original message --------From: Amir-Inbar ***@***.***> Date: 7/20/24  3:34 AM  (GMT-05:00) To: ib-api-reloaded/ib_async ***@***.***> Cc: Subscribed ***@***.***> Subject: [ib-api-reloaded/ib_async] Creating a WhatsApp Group for Sharing Knowledge on Trading with ib_async (Discussion #48) 
Hi everyone,
I'm interested in forming a WhatsApp group dedicated to sharing knowledge and experiences on trading using ib_async. This group will be a platform for developers and traders to discuss strategies, troubleshoot issues, and share insights related to Interactive Brokers API and the ib_async library.
Is it okay to share the WhatsApp group link here?
Thanks!

**Reply by Amir-Inbar** [reply]
Can you send me a link?

**Comment by gnzsnz** [comment]
checkout this one https://groups.io/g/ib-async/topics

**Comment by vasiche** [comment]
https://discord.com/invite/22jB7QtVhj
